### PR TITLE
Casperson/wt 541

### PR DESCRIPTION
**Pull request by casperson**

### CODE REVIEW

File: src/multi-selector.html [hunk 1]
```diff
@@ -2,15 +2,15 @@
 <div class="cp-multi-selector" ng-if="!icon">
 	<input type="input" class="cp-multi-selector__hidden-input" ng-focus="displayDialog()">
 	<div ng-click="displayDialog()" type="text" class="cp-multi-selector__main-input cps-form-control">
-		<div class="cp-multi-selector__pill cps-white cps-bg-gray-10" ng-repeat="item in formatName()">
-			{{item.formatted_name}}<i ng-click="removeItem(item)" class="cps-icon cps-icon-sm-close"></i>
+		<div class="cp-multi-selector__pill cps-white cps-bg-gray-10" ng-repeat="item in selectedItems">
```
Comment (kentmclean) [inline src/multi-selector.html]: You may wish to add sorting in here. "item in selectedItems | orderBy: 'display_name'"

File: src/multi-selector.html [hunk 2]
```diff
@@ -2,15 +2,15 @@
...
+			{{item.display_name}}<i ng-click="removeItem(item)" class="cps-icon cps-icon-sm-close"></i>
 		</div>
 	</div>
 </div>
 <!-- icon mode  -->
 <div ng-if="icon" class="cp-multi-selector__icon">
-	<div ng-if="!dontShowPill" class="cp-multi-selector__pill +inline cps-white cps-bg-gray-10" ng-repeat="item in formatName()">
-		<span style="vertical-align: top">{{item.formatted_name}}</span><i ng-click="removeItem(item)" class="cps-icon cps-icon-sm-close"></i>
+	<div ng-if="!dontShowPill" class="cp-multi-selector__pill +inline cps-white cps-bg-gray-10" ng-repeat="item in selectedItems">
```
Comment (kentmclean) [inline src/multi-selector.html]: same here

File: src/multi-selector.js [hunk 3]
```diff
@@ -47,11 +48,21 @@ angular.module('cp-multi-selector')
...
 				 */
-				scope.formatName = function() {
-					return scope.selectedItems.map((item) => {
-						item.formatted_name = (item.first_name && item.last_name) ? `${item.first_name} ${item.last_name}` : item.name;
-						return item;
-					})
+				function formatName() {
+					if(!scope.displayName){
+						if(scope.selectedItems){
+							scope.selectedItems.map((item) => {
```
Comment (kentmclean) [inline src/multi-selector.js]: You might consider creating a sub-function for this functionality that takes as its argument an array. Then you can pass either scope.selectedItems or scope.source to it, call .map on whatever one it is, and consolidate this a bit.

Comment (casperson) [inline src/multi-selector.js]: And this will reference and modify the original array that is passed in, right?

**Pull request closed without merging**